modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/to/PodcastGrouping.kt

```diff
@@ -5,15 +5,14 @@ import android.content.res.Resources
 import androidx.annotation.StringRes
 import au.com.shiftyjelly.pocketcasts.models.entity.Podcast
 import au.com.shiftyjelly.pocketcasts.models.entity.PodcastEpisode
-import au.com.shiftyjelly.pocketcasts.models.to.PodcastGrouping.Season.getSeasonGroupId
 import au.com.shiftyjelly.pocketcasts.models.type.EpisodesSortType
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 sealed class PodcastGrouping(
     @StringRes val groupName: Int,
     val index: Int,
     val serverId: Int,
-    val sortFunction: ((PodcastEpisode) -> Int)?,
+    var sortFunction: ((PodcastEpisode) -> Int)?,
```

Comment:
Property shouldn't be changed to be mutable only to set it inside the class.

Reply:
[273b798](https://github.com/Automattic/pocket-casts-android/pull/2780/commits/273b7987d9450e9a351dcbb92bf8263150ca8b5d)